### PR TITLE
Add RemoveMember.cs, make AddMember a static method

### PR DESCRIPTION
**Pull request by mdejong3**
I'm updating this doc (https://cloud.google.com/iam/docs/granting-changing-revoking-access) and thought it could benefit from code samples for removing members using the client libraries, so I wrote a RemoveMember.cs code sample.

I also made AddMember a static method to match all of the other methods in this sample.

### CODE REVIEW

**Comment by mdejong3** [comment]
@googlebot I signed it!

File: iam/api/Access/RemoveMember.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+// limitations under the License.
+
+// [START iam_modify_policy_remove_member]
+
+using System.Linq;
+using Google.Apis.CloudResourceManager.v1.Data;
+
+public partial class AccessManager
+{
+    public static Policy RemoveMember(Policy policy, string role, string member)
```
Comment (SurferJeffAtGoogle) [inline iam/api/Access/RemoveMember.cs]: Please add a test that exercises the new code.

Comment (mdejong3) [reply iam/api/Access/RemoveMember.cs]: Added tests

**Comment by mdejong3** [comment]
I haven't really pushed to GitHub much--could you help me understand why these checks are failing and what I need to do to fix it?

Thanks!

**Comment by mdejong3** [comment]
Ready to be checked again!

**Comment by mdejong3** [comment]
I think the windows tests are failing due to linting, but the linux test is still failing on AccessTest--which is unfortunate, because it's still working on my machine. I put in some additional conditions that should give more informative errors, but I can't find them on the invocation log. Would you be able to run the test on your machine and tell me what error it gives you?

**Comment by SurferJeffAtGoogle** [comment]
The test is failing with a permissions issue: 
Failed   GoogleCloudSamples.AccessTest.TestAccess
Error Message:
 Google.GoogleApiException : Google.Apis.Requests.RequestError
The caller does not have permission [403]
Errors [
        Message[The caller does not have permission] Location[ - ] Reason[forbidden] Domain[global]
]

**Comment by mdejong3** [comment]
...oops. I left my test project ID in the _project variable. I removed it--should (hopefully) work now, or at least give a more informative error.

**Comment by mdejong3** [comment]
This is still passing for me, locally. Any idea what's going wrong?

**Pull request merged**